src/sentry/tasks/integrations/github/constants.py

```diff
@@ -0,0 +1,58 @@
+ISSUE_LOCKED_ERROR_MESSAGE = "Unable to create comment because issue is locked."
```

Comment:
I'm just a drive by -- but I think this is worse than before.  when they're directly inline it is very easy to see what text / etc. is going to be used

constants are really only useful if they're used more than once and now I have to hunt in another file to figure out what the actual code is doing instead of it being directly inline

Reply:
i can move some back, but this one is actually used in multiple files